edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/KubeUtils.cs

```diff
@@ -0,0 +1,198 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes
+{
+    using System;
+    using System.Linq;
+    using System.Text;
+    using CoreConstants = Microsoft.Azure.Devices.Edge.Agent.Core.Constants;
+
+    public static class KubeUtils
+    {
+        const string Alphabet = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
+        const string AllowedCharsLabelValues = "-._";
+        const string AllowedCharsDns = "-";
+        const string AllowedCharsGeneric = "-.";
+        const int MaxK8SValueLength = 253;
+        const int MaxDnsNameLength = 63;
+        const int MaxLabelValueLength = 63;
+
+        static bool IsAlpha(char ch) => Alphabet.IndexOf(ch) != -1;
+
+        static bool IsAlphaNumeric(char ch) => IsAlpha(ch) || Char.IsDigit(ch);
+
+        public static readonly string K8sNamespace = Constants.k8sNamespaceBaseName;
```

Comment:
Why do we need this string? Couldn't we just use `Constants.k8sNamespaceBaseName`?

Reply:
Ah, it is re-assigned on line 35.

Reply:
Actually, I'm pulling this out, because the namespace will need to be passed in to anything that needs it.